ios/FluentUI/Controls/SearchBar.swift

```diff
@@ -119,6 +120,15 @@ open class SearchBar: UIView {
                 return Colors.SearchBar.DarkContent.tint
             }
         }
+
+        func progressSpinnerColor(for window: UIWindow) -> UIColor {
+            switch self {
+            case .lightContent:
+                return UIColor(light: Colors.primaryTint10(for: window), dark: Colors.iconSecondary)
```

Comment:
Same here about the colors:

It's using gray500 instead of gray100 defined by @Victa 

Light mode / primary (= blue nav bar): Accent color Tint 10
Light mode / white: Gray 300
Dark mode / default: **Gray 100**

@objc public static let iconSecondary = UIColor(light: gray400, lightHighContrast: gray600, dark: **gray500**, darkHighContrast: gray300, darkElevated: gray400)

Reply:
updated colors. 

---
In reply to: [468190424](https://github.com/microsoft/fluentui-apple/pull/174#discussion_r468190424) [](ancestors = 468190424)